sonic-xcvrd/xcvrd/dom/dom_mgr.py

```diff
@@ -210,6 +251,10 @@ def task_worker(self):
         # Set the periodic db update time
         dom_info_update_periodic_secs = self.DOM_INFO_UPDATE_PERIOD_SECS
 
+        # Wait for all PORTs to be initialized
+        self.wait_port_initialization(dom_info_update_periodic_secs)
+        self.log_notice("All ports are in CMIS terminal state, start DOM monitoring")
+
         # Adding dom_info_update_periodic_secs to allow xcvrd to initialize ports
         # before starting the periodic update
         next_periodic_db_update_time = datetime.datetime.now() + datetime.timedelta(seconds=dom_info_update_periodic_secs)
```

Comment:
@prgeor Wondering if we should set is_periodic_db_update_needed to True since all the ports would have been in CMIS terminal state by this point?